sql-cli/sql_cli/project.py

```diff
@@ -168,11 +206,18 @@ def load_config(self, environment: str = DEFAULT_ENVIRONMENT) -> None:
         config = Config(environment=environment, project_dir=self.directory).from_yaml_to_config()
         if config.airflow_home:
             self._airflow_home = Path(config.airflow_home).resolve()
+        else:
+            self._initialise_env_config(environment)
+            config = Config(environment=environment, project_dir=self.directory).from_yaml_to_config()
+
+        if not self.get_env_airflow_home(environment).exists():
+            self._initialise_airflow_for_environment(environment)
+
```

Comment:
Why did we add this? This should be done by `initialise`.

Reply:
The following use case fails if we don't handle this afterwards:
```
$ flow init .
# user manually adds a new environment `stage` to the `config` folder
$ flow validate --env stage
```

I realised this use case with one of the existing tests!

Reply:
Right! Thanks 👍 Can we simplify this?

Reply:
@feluelle I added comments - how would you suggest simplifying it?